src/components/Additem.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function additem() {
```

Comment:
컴포넌트의 경우 맨 앞글자가 대문자여야합니다!
```suggestion
function Additem() {
```